<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSRMainWindow</class>
 <widget class="QMainWindow" name="RSRMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSRMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="m_tableView">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>781</width>
      <height>291</height>
     </rect>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QWebView" name="m_webView">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>339</y>
      <width>800</width>
      <height>461</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="m_mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="m_statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
